virt-who-config.5

```diff
@@ -249,6 +249,27 @@ Absolute path to the JSON file that has the same structure as file returned from
 .br
 }
 
+.SH CONFIGURATION MIGRATION
+
+Previous versions of virt-who employed other configuration means based on environment variables. The first was system-wide environment variables. The second was a file [/etc/sysconfig/virt-who]
+which contained variables to be used on a session basis when run as a service.
+
+In an effort to simplify the process, and to bring consistency to virt-who's execution regardless of initiation [service vs. command line], the configuration of the application has been
+consolidated into the general configuration file as outlined above. To update previous versions of the configuration layout, a migration script has beeen included. That script will incorporate
+the known system environment variables [VIRTWHO_INTERVAL, VIRTWHO_DEBUG, VIRTWHO_ONESHOT] and the entries in the /etc/sysconfig/virt-who file into the general configuration file. The known variables
+will land in the \fBglobal\fR section while any others in the /etc/sysconfig/virt-who [i.e. HTTPS_PROXY] will land in the \fBsystem_environment\fR section. This migration may result
+in multiple entries for a specific fields, but the priority order will be maintained so that the behavior will remain constant.
+
+Each new entry in the general configuration file will come after a comment indicating that it was migrated. The /etc/sysconfig/virt-who file will be deleted after its entries are
+migrated, but the known system environment variables  will need to be manually removed or they will be migrated again if the script is rerun. Those variables will not be recognized
+by virt-who even if they remain.
+
+The spec file will cause the migration script to run when the RPM is installed, but you can run it manually with python after the RPM is installed:
```

Comment:
I would not mention spec file here, because administrator of systems needn't to have experience with creation of RPM and referencing .spec file could be confusing. I would write here something like: 'The migration script will be run, when the RPM is installed, but ...'